install.sh

```diff
@@ -1,20 +1,6 @@
 #!/usr/bin/env bash
-function doInstall {
-	if ! [ -L ~/bin ] && ! [ -d ~/bin ]; then
-		echo make dir
-		mkdir ~/bin
-	else
-		echo dir '~/bin' exist
-	fi
-	cd /tmp && \
-	git clone https://github.com/imachug/win-sudo.git && \
-	mv win-sudo/su* ~/bin && \
-	rm -rf win-sudo
-}
-
-function doFail {
-	echo "failed to install, sorry :'(";
-	exit 1
-}
-doInstall || doFail
-echo "enjoy :)"
+echo "Downloading win-sudo...";trap 'echo "Failed to install, sorry :(";exit 1' ERR
+mkdir -p ~/bin/win-sudo && cd "$_";git init -q && git config core.sparsecheckout true;echo s/ >> .git/info/sparse-checkout
+git remote add -mf origin https://github.com/DemonixCraft/win-sudo.git && git pull -q origin master
```

Comment:
Can you please use `imachug` instead?

Reply:
Yeah I'll put some changes in a new commit